src/head.ts

```diff
@@ -46,6 +54,12 @@ const getTagsData = (appUrl: string): Tag[] => [
             href: `${appUrl}${webManifest}`,
         },
     },
+    {
+        tag: 'link',
+        attribs: {
+            ...fontAttribs,
+        },
```

Comment:
Kan droppe spread'en
```suggestion
        attribs: fontAttribs,
```